master/test/streamdata/stream_data.go

```diff
@@ -66,55 +68,14 @@ type trial struct {
 
 // ModTrial is a convenience function for modifying rows of a trial.
 func ModTrial(
-	ctx context.Context, trialID, experimentID int, changeStart bool,
-	changeSeq bool, changeState string,
+	ctx context.Context, newTrial Trial,
```

Comment:
question: is it even worth it to keep this function if all it is doing is a bun query? I'm thinking that it could help save the actual testing code from a bit of bulk, but this is barely worthy of a function tbh

Reply:
I really don't have much of a preference one way or the other, if you see it reducing significant bulk then I'd say keep it; otherwise, in line it and get rid of the function overhead

Reply:
I'll keep it as is, I like centralizing the request elsewhere and trying to abstract it out of the testing code as much as possible

Reply:
Sounds good to me! Keeping the test code as clear as possible is huge plus!